pywhat/Data/regex.json

```
@@ -333,6 +333,19 @@
          "Bug Bounty"
       ]
    },
+   {
+      "Name": "NPM Token",
```

In your personal opinion, should we rename it to:

```suggestion
      "Name": "Node Package Manager (NPM) Token",
```

? Or is NPM so popular that people know it by acronym?